### ISSUE TITLE
Window is completely white

### ISSUE THREAD

**Issue by Eschguy** [rustdesk/rustdesk#6756]
### Bug Description

Opening up RustDesk, the window is completely white. Sometimes restarting allows the main window to open normally, but trying to remote into an endpoint results in a white desk.

### How to Reproduce

Trying to load an installed version or running the .exe fresh.

### Expected Behavior

Window is visible, usable, and remote session windows are visible and interactable.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10/11

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/8880292/85e00abb-55c2-4c7f-950c-bc7610ce7a05)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/5108

**Comment by Morzorz** [comment]
你是远程安装的吧

**Comment by Eschguy** [comment]
> #5108 [#5108 (comment)](https://github.com/rustdesk/rustdesk/issues/5108#issuecomment-1837335668)

Per the recommendations in those threads, I tried the sciter version and it doesn't seem to be doing it. Will use it going forward until a fix is found for the flutter version.

**Comment by ugenehan** [comment]
> Per the recommendations in those threads, I tried the sciter version and it doesn't seem to be doing it. Will use it going forward until a fix is found for the flutter version

me too,also with sciter version.
![image](https://github.com/rustdesk/rustdesk/assets/9714462/1e022804-b3ff-44cd-9737-b7010c6975e2)

**Comment by JSoko** [comment]
Hello, I noticed this white window in connection with Windows Powertoys. The workaround is to start Rustdesk and then Powertoys

**Comment by ugenehan** [comment]
> Powertoys

There is no Powertoys on my system.

**Comment by ugenehan** [comment]
> Did you install it remotely?
> 
> > 你是远程安装的吧

This is my reason.

**Comment by SkysCrystal** [comment]
Installed remotely. On Windows 7. 
Rustdesk version 1.2.3 / 1.2.4 are blank:
![2024-01-09_173525](https://github.com/rustdesk/rustdesk/assets/49473463/f7ef6361-f10b-49e8-8cc8-7c563408dcf4)

Rustdesk version 1.1.8 execute properly, but can not be conneted with Rustdesk version 1.2.3.
![2024-01-09_173913](https://github.com/rustdesk/rustdesk/assets/49473463/a2e77dc5-4109-45d5-840e-bba22f6bfc52)

**Comment by rustdesk** [comment]
> Rustdesk version 1.1.8 execute properly, but can not be conneted with Rustdesk version 1.2.3.

You can try out the Sctier x86 version of 1.2.3 / 1.2.4. rustdesk-1.2.4-x86-sciter.ex

**Comment by SkysCrystal** [comment]
> > Rustdesk version 1.1.8 execute properly, but can not be conneted with Rustdesk version 1.2.3.
> 
> You can try out the Sctier x86 version of 1.2.3 / 1.2.4, this version uses the same tech stack as 1.1.8. `rustdesk-1.2.4-x86-sciter.exe`

Thank you! Problem is solved.

**Comment by domy86** [comment]
I have excluded rustdesk.exe in PowerToys FancyZones, restarted RustDesk service and works fine so far!

**Comment by vkonstad** [comment]
> I have excluded rustdesk.exe in PowerToys FancyZones, restarted RustDesk service and works fine so far!
> 
> Windows 11 to Windows 10 connection, RustDesk 1.2.2 on both ends.

Thank you thank you thank you. That worked for me too!!

**Comment by domy86** [comment]
@vkonstad, exactly, figured that out later after I've experienced some issues with pop-up windows inside MS Office.

@Eschguy, how is going on your side?

**Comment by Eschguy** [comment]
Disabling that setting in Fancy Zones seems to have fixed the issue for me as well!

**Comment by Godor** [comment]
所有的版本都有这个情况，我现在有一台电脑，下载了所有版本，打开都是白屏。实际页面是加载了的。由于无法重现，我电脑是常年开机的，可以给你远程协助，你检测一下。@rustdesk

**Comment by 21pages** [comment]
@Godor Please contact me with cGFnZXMyMUAxNjMuY29t, base64

**Comment by 21pages** [comment]
@Godor reported that the computer, which has no monitor and is not shut down for a long time, experienced issues with both Sciter and Flutter. However, the issue was resolved later, possibly due to the insertion of a monitor. The problem could not be reproduced by plugging and unplugging the monitor. Another related issue is that when the main window is present, disabling and enabling the driver causes the window to become transparent. This is a known issue in Flutter, https://github.com/flutter/flutter/issues/124194#issuecomment-1498632861.

**Comment by sesu-tech** [comment]
> Hello, I noticed this white window in connection with Windows Powertoys. The workaround is to start Rustdesk and then Powertoys

Same issue here, stopping PowerToys fixed it for me as well!

**Comment by AwakeningLightning** [comment]
Same issue with Visual Studio, screen is blank on remote windows, I'm accessing from Linux

**Comment by chansheunglong** [comment]
Can also confirm disabling PowerToys' "Move newly created windows to their last known zone" will solve the issue on WIndows 11 23H2 with PowerToys v0.81.1

**Comment by sanebg** [comment]
Same issue, can't believe this simple issue is not fixed for so long!

**Comment by jan-the-maker** [comment]
> Can also confirm disabling PowerToys' "Move newly created windows to their last known zone" will solve the issue on WIndows 11 23H2 with PowerToys v0.81.1

I think just adding RustDesk in the last windows of Fancy Zone where you can add the exclusions has the same effect. For me it seems to work.

**Comment by vcan** [comment]
![Xnip2024-08-13_21-16-52](https://github.com/user-attachments/assets/4ad568fa-fe87-491d-aafc-16291becabcf)
mac os system when lock the screen reopen, the ruskdesk is black.
mac os version 13.3.1 (22E261)

**Comment by 21pages** [comment]
Can you run the program below? Unzip, click rustdesk.exe, and a console will pop up. IWhen the interface displays abnormally, paste the console content here.

https://github.com/21pages/rustdesk/releases/download/tmp/rustdesk_flutter_debug_console.zip

**Comment by CrPO4** [comment]
@21pages 
Not the original poster but can confirm this issue is produced by PowerToys' FancyZone functionality. I presume that every tool that does the same that tool does is going to do that to Rustdesk.
Here's "debug" for NORMAL run with Powertoys running BUT rustdesk.exe exlcluded:

`[2024-09-20T08:29:06Z INFO  librustdesk::lan] lan discovery listener started
flutter: The Dart VM service is listening on http://127.0.0.1:56308/Y6AMVuDB7KU=/
flutter: launch args: []
flutter: initializing FFI main
flutter: _appType:main,info1-id:XXX.YYY.ZZZ,info2-name:XXX.YYY.ZZZ,dir:C:\Users\CrPO4\Documents
flutter: _globalFFI init
flutter: _globalFFI init end
flutter: registerEventHandler native_ui native_ui
flutter: registerEventHandler callback_query_onlines favorite peer
flutter: registerEventHandler load_fav_peers favorite peer
flutter: restore lpos: 2568.0/1448.0, offset:-1.0/0.0, devicePixelRatio:1.0, isMaximized: false, isFullscreen: false
flutter: restore new offset: -1.0/0.0, scale:1.0
flutter: initialLink:
flutter: handled by uni links: false
flutter: [MultiWindowHandler] active window changed: {0}
[2024-09-20T08:29:08Z INFO  scrap::common::hwcodec] set hwcodec config
[2024-09-20T08:29:08Z DEBUG scrap::common::hwcodec] HwCodecConfig { signature: 18014403646693497, ram_encode: [CodecInfo { name: "h264_nvenc", mc_name: None, format: H264, priority: 0, hwdevice: AV_HWDEVICE_TYPE_NONE }, CodecInfo { name: "hevc_nvenc", mc_name: None, format: H265, priority: 0, hwdevice: AV_HWDEVICE_TYPE_NONE }, CodecInfo { name: "h264_amf", mc_name: None, format: H264, priority: 0, hwdevice: AV_HWDEVICE_TYPE_NONE }, CodecInfo { name: "hevc_amf", mc_name: None, format: H265, priority: 0, hwdevice: AV_HWDEVICE_TYPE_NONE }], ram_decode: [CodecInfo { name: "h264", mc_name: None, format: H264, priority: 0, hwdevice: AV_HWDEVICE_TYPE_D3D11VA }, CodecInfo { name: "hevc", mc_name: None, format: H265, priority: 0, hwdevice: AV_HWDEVICE_TYPE_D3D11VA }, CodecInfo { name: "h264", mc_name: None, format: H264, priority: 3, hwdevice: AV_HWDEVICE_TYPE_NONE }, CodecInfo { name: "hevc", mc_name: None, format: H265, priority: 3, hwdevice: AV_HWDEVICE_TYPE_NONE }], vram_encode: [FeatureContext { driver: AMF, luid: 35351847, api: API_DX11, data_format: H264 }, FeatureContext { driver: AMF, luid: 35351847, api: API_DX11, data_format: H265 }, FeatureContext { driver: FFMPEG, luid: 65467, api: API_DX11, data_format: H264 }, FeatureContext { driver: FFMPEG, luid: 35351847, api: API_DX11, data_format: H264 }, FeatureContext { driver: FFMPEG, luid: 65467, api: API_DX11, data_format: H265 }, FeatureContext { driver: FFMPEG, luid: 35351847, api: API_DX11, data_format: H265 }, FeatureContext { driver: NV, luid: 65467, api: API_DX11, data_format: H264 }, FeatureContext { driver: NV, luid: 65467, api: API_DX11, data_format: H265 }], vram_decode: [DecodeContext { device: None, driver: AMF, luid: 35351847, api: API_DX11, data_format: H264 }, DecodeContext { device: None, driver: FFMPEG, luid: 65467, api: API_DX11, data_format: H264 }, DecodeContext { device: None, driver: FFMPEG, luid: 35351847, api: API_DX11, data_format: H264 }, DecodeContext { device: None, driver: FFMPEG, luid: 65467, api: API_DX11, data_format: H265 }, DecodeContext { device: None, driver: FFMPEG, luid: 35351847, api: API_DX11, data_format: H265 }] }
[2024-09-20T08:29:08Z DEBUG reqwest::async_impl::client] redirecting 'https://github.com/rustdesk/rustdesk/releases/latest' to 'https://github.com/rustdesk/rustdesk/releases/tag/1.3.0'
[2024-09-20T08:29:08Z INFO  scrap::common::hwcodec] Check hwcodec config, exit with: exit code: 0`

And here is the "bugged" run without an exclusion:
`[2024-09-20T08:32:11Z INFO  librustdesk::lan] lan discovery listener started
flutter: The Dart VM service is listening on http://127.0.0.1:56345/cvRk6gfcAMg=/
flutter: launch args: []
flutter: initializing FFI main
flutter: _appType:main,info1-id:Ei.horne.lan,info2-name:Ei.horne.lan,dir:C:\Users\CrPO4\Documents
flutter: _globalFFI init
flutter: _globalFFI init end
flutter: registerEventHandler native_ui native_ui
flutter: registerEventHandler callback_query_onlines favorite peer
flutter: registerEventHandler load_fav_peers favorite peer
flutter: restore lpos: 2568.0/1448.0, offset:250.0/370.0, devicePixelRatio:1.0, isMaximized: false, isFullscreen: false
flutter: restore new offset: 250.0/370.0, scale:1.0
flutter: initialLink:
flutter: handled by uni links: false
flutter: [MultiWindowHandler] active window changed: {0}
[2024-09-20T08:32:13Z DEBUG reqwest::async_impl::client] redirecting 'https://github.com/rustdesk/rustdesk/releases/latest' to 'https://github.com/rustdesk/rustdesk/releases/tag/1.3.0'
[2024-09-20T08:32:41Z INFO  scrap::common::hwcodec] Check hwcodec config, exit with: exit code: 1`

**Comment by 21pages** [comment]
This doesn't happen on my computer.  In the bugged log,  it seems the hwcodec check is halted. Could you please try disabling the hwcodec option and then restart RustDesk to see if the white screen issue persists?

https://github.com/user-attachments/assets/fe45f23f-82c0-44b6-acd2-2722f6afee25

**Comment by CrPO4** [comment]
@21pages 
Just tried it - "hardware codec" option doesn't make a difference in my case.

**Comment by 21pages** [comment]
Okay, still most likely related to flutter.

**Comment by CrPO4** [comment]
@21pages 
This one works just fine, no matter what's enabled in FancyZones and no matter the previous window state.

**Comment by rustdesk** [comment]
Powertoy + Flutter:

https://github.com/flutter/flutter/issues/137321#issuecomment-1780946083
https://github.com/flutter/flutter/issues/149709#issuecomment-2164040892

Flutter team seems no interest to fix it.

**Comment by CrPO4** [comment]
> Powertoys + Flutter:
> 
> [flutter/flutter#137321 (comment)](https://github.com/flutter/flutter/issues/137321#issuecomment-1780946083) [flutter/flutter#149709 (comment)](https://github.com/flutter/flutter/issues/149709#issuecomment-2164040892)
> 
> Google's Flutter team seems no interest to fix it.

Such a shame, guess we'd have to stick to workaround then (either exclusion for Rustdesk or disabling "Move newly created windows to their last known zone" in FancyZones) until Google decides to take action.

**Comment by rustdesk** [comment]
Flutter 3.24 fixed some blank window issues, we will upgrade in next version, not sure if it can fix powertoys. https://github.com/flutter/flutter/issues/150546

**Comment by RossComputerGuy** [comment]
Any timeline on updating to Flutter 3.24? We're looking to drop Flutter 3.19 in Nixpkgs so everything can compile on 64k page size systems under Flutter 3.24.

**Comment by Jacky-YC** [comment]
![image](https://github.com/user-attachments/assets/6446b843-9854-4592-ba7a-2c4e0f67b476)


sciter 不正常

**Comment by rustdesk** [comment]
> Flutter 3.24 fixed some blank window issues, we will upgrade in next version, not sure if it can fix powertoys. [flutter/flutter#150546](https://github.com/flutter/flutter/issues/150546)

We upgraded to 3.24, please try it out https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Comment by TryingToDoBetter25** [comment]
> > Flutter 3.24 fixed some blank window issues, we will upgrade in next version, not sure if it can fix powertoys. [flutter/flutter#150546](https://github.com/flutter/flutter/issues/150546)
> 
> We upgraded to 3.24, please try it out https://github.com/rustdesk/rustdesk/releases/tag/nightly

Thank you for the great work. Unfortunately, I have tried rustdesk-1.33-x86_64.exe and rustdesk-1.34-x86-script.exe, but they still haven't solved the problem of the white window for me.

**Comment by CrPO4** [comment]
Unfortunately, I see the same behavior on the nightly too.

**Comment by TryingToDoBetter25** [comment]
I found that in version rustdesk-1.33-x86_64.exe, if the other host has a display device, the white window issue disappears. If there is no display device, the white window issue still exists. I hope this is helpful to developers.

**Comment by dongboyu1114** [comment]
discovered this issue because, when the controlled device is not connected to or has disconnected from a monitor, opening RustDesk while using other software like ToDesk results in a white screen in RustDesk. This problem occurs with both versions of RustDesk. It doesn't seem to be related to the Flutter version, as I’ve also tried another version. Perhaps it's caused by the controlled device not being connected to an external monitor.

**Comment by LookAtMyC** [comment]
@rustdesk  DM me if you need a VM with the blank screen issue to remote connect and tinker on that issue

**Comment by CainLuo** [comment]
I have the same problem. Whether it is .msi or .exe, it is a blank interface.

System: Windows 11 24H2 Pro
CPU: AMD 7500F
RustDesk Version: 1.3.3

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/86b583d0-f688-4485-b6fd-772e9ca5c16d">

**Comment by rustdesk** [comment]
Could you try `rustdesk.exe --install` from command line? if it is blank.

**Comment by 21pages** [comment]
@LookAtMyC @CainLuo 

Thank you for your feedback. Could you please test the simplest Flutter demo below? This will help us determine if the issue is related to Flutter.

[flutter_simple_demo.zip](https://github.com/user-attachments/files/17956140/flutter_simple_demo.zip)

**Comment by CrPO4** [comment]
@21pages 
I made a video that showcases this problem.
https://www.youtube.com/watch?v=hr3hcWKMIhY

**Comment by rustdesk** [comment]
> @21pages I made a video that showcases this problem. https://www.youtube.com/watch?v=hr3hcWKMIhY It is 100% reproducible with this demo.

The demo can be blank, we have no ability to fix Flutter. How about the RustDesk 32bit version?

**Comment by rustdesk** [comment]
> @21pages I made a video that showcases this problem. https://www.youtube.com/watch?v=hr3hcWKMIhY It is 100% reproducible with this demo.

@CrPO4 Could you submit this issue to Flutter team?

**Comment by CrPO4** [comment]
> > @21pages I made a video that showcases this problem. https://www.youtube.com/watch?v=hr3hcWKMIhY It is 100% reproducible with this demo.
> 
> Wow, the demo is blank, we have no ability to fix Flutter. How about the RustDesk 32bit version?

@rustdesk 
As before, it's not reproducible with Sciter versions:
https://youtu.be/W-GB7_fDHE0

Will probably file an issue to Flutter later today but it'd be much better if someone with more knowledge did since I won't really be able to provide much info to them.

**Comment by rustdesk** [comment]
> provide much info to them

We can not either. We filed before, they did nothing. But I think if more people file, they may consider, hope so.

**Comment by CrPO4** [comment]
@rustdesk 
Alright then, will try my best to raise this to their attention.
Let's hope they fix this at some point.

**Comment by CrPO4** [comment]
Filed an issue to Flutter, I hope it's good enough for them.
https://github.com/flutter/flutter/issues/159630

**Comment by DannyDeLoach** [comment]
Can someone summarize available workarounds, while we wait for the Flutter team to fix things on their end?

**Comment by Chaoscontrol** [comment]
> Can someone summarize available workarounds, while we wait for the Flutter team to fix things on their end?

The ones I know. 
- Close PowerToys. Open Rustdesk first, then Powertoys. 
- Exclude `rustdesk.exe` in PowerToys FancyZones. This is what I do.

**Comment by CainLuo** [comment]
> @LookAtMyC @CainLuo
> 
> Thank you for your feedback. Could you please test the simplest Flutter demo below? This will help us determine if the issue is related to Flutter.
> 
> [flutter_simple_demo.zip](https://github.com/user-attachments/files/17956140/flutter_simple_demo.zip)

@21pages  Sorry, I forgot to tell you the test results. It’s the same white page, so it should be the same problem.

**Comment by gaojian88888888** [comment]
@rustdesk When I encounter this issue (after connecting via RustDesk, the client-side RustDesk screen turns white, and some windows display a blank screen), I first use RustDesk to open another remote access tool (e.g., UU Remote) on the controlled device. I connect via UU Remote, then disconnect it. Afterward, reconnecting with RustDesk resolves the problem. What’s the underlying reason for this?versions：1.3.8

**Comment by mrhan1993** [comment]
In the situations I've encountered, when using remote tools (such as Sunlogin / even including RustDesk), this issue may occur (not necessarily), while there is no such problem when using mstsc.

**Comment by tableking** [comment]
When I use rust-desk for 2 days, it occurs to my machine. All kinds of 1.4.0 exe version are tried, it has the blank windows, still.
**client-side | win 10 x64 | 1.4.0**

**Comment by wxyShine** [comment]
I have tried many versions including sciter，Same result, white screen when running the software.     Do you need to install a PowerToys and then exclude rustdesk.exe from it?

**Comment by SixFive7** [comment]
Still an issue on the latest version of Windows, Powertoys and Rustdesk per 2025-10-14. For everybody affected [this workaround](https://github.com/rustdesk/rustdesk/issues/6756#issuecomment-1916496426) still works.

**Comment by Kurgan-** [comment]
Rustdesk 1.4.2, not installed, just launched with configuration data and the string "qs" in file name (so that it requests elevation) on Win 11 pro 25h2 freshly installed, no powertoys and no "strange" software installed (just office, libreoffice, chrome, firefox) results in a white window.  (It asks for elevation, you allow it, you get a white window so you cannot read the desk ID to the remote party). If you minimize the window and then un-minimize it, it works. 

To be perfectly clear: this is not a remote control issue, the rustdesk main window is white locally, before connection.

**Comment by UNeedCryDear** [comment]
This issue can be 100% reproduced on machines without a physical monitor. If the settings are properly configured, the system can function normally without any impact—the only problem is that RustDesk shows a blank screen, while the remote access functionality itself works fine.
My solution to this is to use a graphics card output emulator (available for HDMI, DP, or VGA interfaces). This allows for normal remote access without having to turn on a physical monitor.

**Comment by Kurgan-** [comment]
> This issue can be 100% reproduced on machines without a physical monitor. 

Uhm. This is interesting. The machine I was remoting in surely has a monitor, there was the customer in front of it. I don't know if it has TWO monitors and if that's the issue.

**Comment by rustdesk** [comment]
@Kurgan-  could you test this demo? https://github.com/rustdesk/rustdesk/issues/6756#issuecomment-2507280693

**Comment by JohnAdstra** [comment]
> ### Bug Description
> Opening up RustDesk, the window is completely white. Sometimes restarting allows the main window to open normally, but trying to remote into an endpoint results in a white desk.
> 
> ### How to Reproduce
> Trying to load an installed version or running the .exe fresh.
> 
> ### Expected Behavior
> Window is visible, usable, and remote session windows are visible and interactable.
> 
> ### Operating system(s) on local side and remote side
> Windows 10 -> Windows 10/11
> 
> ### RustDesk Version(s) on local side and remote side
> 1.2.3
> 
> ### Screenshots
> ![image](https://github.com/rustdesk/rustdesk/assets/8880292/85e00abb-55c2-4c7f-950c-bc7610ce7a05)
> 
> ### Additional Context
> _No response_

Yes. Am having issues with a new Pi 5 running Bookworm also. The only solid connection I can make from any remote desktop is using Raspberry Pi Connect with the Wayland backend. From a Windows 11 PC, a RustDesk connection is made but the screen is black (LAN). NoMachine yields the following error: The remote host '10.0.0.xx' refused to establish a network connection on port '4000'. Please verify that NoMachine is correctly installed and listening on the port where you are trying to connect, and that incoming traffic is not being blocked by a firewall or NAT. AnyDesk will not launch.  If I switch to the X11 backend, RPC does not work, and RustDesk sometimes works but not all the time. 

Would like to send this Pi5 to a remote site to collect scientific data, and RPC does not have file transfer as needed. Furthermore there are already several Pi4bs on site and would like to be able to do screen sharing back and forth. Virtually all these apps work with the Pi4s, and not with the Pi5. Since the 5's have been out for some time now, would hope someone would address these issues. Any thoughts appreciated. Thanks.

**Comment by Kurgan-** [comment]
@JohnAdstra Your issue seems to be different. You get a blank window once you are connected to the remote. We get a blank window as soon as we start rustdesk, before connecting. We launch rustdesk and see a blank window, no user interface, no way even to start a connection. (and our issue seems to be non reproducible, happens only sometimes and only on some windows computers, not on others)

**Comment by JohnSmith436** [comment]
> [@JohnAdstra](https://github.com/JohnAdstra) Your issue seems to be different. You get a blank window once you are connected to the remote. We get a blank window as soon as we start rustdesk, before connecting. We launch rustdesk and see a blank window, no user interface, no way even to start a connection. (and our issue seems to be non reproducible, happens only sometimes and only on some windows computers, not on others)

Had the same issue on my computer when RustDesk was installed and tun under a user account.
Opened either a white blank window, or just an outline (frame) of a window. Worked fine if launched under Admin account.
After I read here PowerToys might be the issue, tried the following:
I quit powertoys, launched RustDesk, It launched normally without issues. 
I quit RD, launched powertoys, and launched RustDesk afterwards, it launched normally this way as well.
Added it to exceptions in powertoys but as it is my main machine, haven't had time to test if it works normally after a reboot though.

**Comment by rekonnected** [comment]
Confirming this, 100% reproduction when *no monitor is active.* This includes a laptop set to remain unlocked but turn the display off when the lid is closed. Opening the laptop and moving the mouse within the window cleared the rendering up immediately. I do not have PowerToys or any other special tools of the like installed.

> This issue can be 100% reproduced on machines without a physical monitor. If the settings are properly configured, the system can function normally without any impact—the only problem is that RustDesk shows a blank screen, while the remote access functionality itself works fine. My solution to this is to use a graphics card output emulator (available for HDMI, DP, or VGA interfaces). This allows for normal remote access without having to turn on a physical monitor.

**Comment by rubjonny** [comment]
Just had the same issue, its been fine for months then all of a sudden I'm hitting this issue. To work around it I've installed this:
https://github.com/VirtualDrivers/Virtual-Display-Driver

**Comment by Habetdin** [comment]
> Just had the same issue, its been fine for months then all of a sudden I'm hitting this issue. To work around it I've installed this: https://github.com/VirtualDrivers/Virtual-Display-Driver

@rubjonny thank you for the workaround! Someone turned my office PC's monitor off and my RustDesk session suddenly got frozen. I tried reconnecting with no success and even rebooting the remote PC but only got the full-white RustDesk window after the reboot.

I've installed VDD and the RustDesk started to work again, even without the restart! I had to use alternative remote desktop software to do this, though.

@rustdesk I guess this is another case of full-white window when there's no active monitor, just as @rekonnected mentioned above.

**Comment by reyaz006** [comment]
So, because RustDesk does not display anything if physical monitor is not attached, I have to launch setup with `--silent-install` in order to update a remote installation. And then launch rustdesk with `--install-service` in case I was stupid enough to choose **Stop service** from the tray menu.

That's with sciter on Windows 11. So `sciter ok` is false.

**Comment by isaq-ue** [comment]
> I have excluded rustdesk.exe in PowerToys FancyZones, restarted RustDesk service and works fine so far!

This solved it! Thank you!

**Comment by jroachgit** [comment]
> Hello, I noticed this white window in connection with Windows Powertoys. The workaround is to start Rustdesk and then Powertoys

This worked for me! Thanks. I'll try and exclude rustdesk from the Windows PowerToys window manager.

**Comment by chrilep** [comment]
![FancyZones app ignore setting](https://i.vgy.me/okmq0A.png)

Faced this bug today. Putting it on powertoys ignore list for FancyZones solved it.

**Comment by disarticulate** [comment]
am getting shit also. FancyZones. Was working just fine a couple of versions ago.

**Comment by kaelbashir** [comment]
> ![FancyZones app ignore setting](https://camo.githubusercontent.com/84b411fd89dff322890b2458c0c7018d8d33c075944478faf1e8162bf3c5e46c/68747470733a2f2f692e7667792e6d652f6f6b6d7130412e706e67)
> 
> Faced this bug today. Putting it on powertoys ignore list for FancyZones solved it.

this fixed my problem that i have been having since yesterday.

I had tried other suggestions e.g. running in administrator mode, updating to latest, nothing worked until i saw your guidance here. Thank you!!!

**Comment by VictorMaxWang** [comment]
Hi, I’d like to work on the Windows/Flutter + PowerToys FancyZones variant of this issue, specifically the blank RustDesk main window before any remote connection. I am not claiming the separate headless/no-active-monitor reports grouped into this thread.

I’ll first reproduce it from the current master on Windows with FancyZones’ “Move newly created windows to their last known zone” enabled, then trace the Flutter window creation/restoration path. I’ll keep the change narrowly scoped, add focused regression coverage where feasible, and verify launch/relaunch with the setting enabled and disabled, both with and without RustDesk in the FancyZones exclusion list.

Please let me know if someone is already working on this specific variant.

**Comment by VictorMaxWang** [comment]
Update: after posting my claim, I found that PR #15717 had already been merged into `master` and addresses this exact Windows/Flutter + FancyZones startup variant. I had initially searched only open PRs and missed the merged fix.

I’m withdrawing my claim and will not open a duplicate PR. Sorry for the noise.

**Comment by rustdesk** [comment]
@VictorMaxWang please test https://github.com/rustdesk/rustdesk/pull/15717 if it works for you.

@kaelbashir

The binary is ready in nightly build.

https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Comment by wxyShine** [comment]
> [@VictorMaxWang](https://github.com/VictorMaxWang)请测试一下[#15717](https://github.com/rustdesk/rustdesk/pull/15717)是否对您有效。
> 
> [@kaelbashir](https://github.com/kaelbashir)
> 
> 二进制文件已在每日构建版本中准备就绪。
> 
> https://github.com/rustdesk/rustdesk/releases/tag/nightly

我在之前版本白屏的电脑上使用这个版本  正常了
